spec/features/user_registration_spec.rb

```diff
@@ -69,40 +70,40 @@ def signup(username, email, password)
  
 describe "Signing in" do
 
-describe "successfully" do
-  let!(:user) { FactoryGirl.create(:user) } 
-  it "displays a flash message indicating success" do
-    visit user_session_path #visit users/sign_in page
-    fill_in 'Email',    with: user.email
-    fill_in 'Password', with: user.password
-    click_button "Sign In"
-    expect(current_path).to eq "/"
-    expect(page).to have_content("Hello")
+  describe "successfully" do
+    let!(:user) { FactoryGirl.create(:user) } 
+    it "displays a flash message indicating success" do
+      visit user_session_path #visit users/sign_in page
```

Comment:
Also, we can just keep track of all the things we want to refactor and then have a refactoring session next week if you want! :)